deep_quoridor/src/quoridor_env.py

```diff
@@ -134,36 +124,9 @@ def _get_observation(self, agent_id):
         walls = self.walls.copy()
 
         return {
-            "board": board,
-            "walls": walls,
-            "my_walls_remaining": self.walls_remaining[agent_id],
-            "opponent_walls_remaining": self.walls_remaining[self.get_opponent(agent_id)],
+            "game": self._game,
```

Comment:
I'm not sure about the consequences here of returning the game, because PettingZoo expects us to return based on the format in `observation_space` method, that's usually things like `spaces.Discrete(...)`